unit_tests/test_tlslite_constants.py

```diff
@@ -219,3 +219,13 @@ def test_getHash_with_valid_new_name(self):
         ret = SignatureScheme.getHash('rsa_pss_pss_sha384')
 
         self.assertEqual(ret, 'sha384')
+
+    def test_getKeyType_with_valid_new_name(self):
+        ret = SignatureScheme.getKeyType('rsa_pss_pss_sha384')
+
+        self.assertEqual(ret, 'rsa')
+
+    def test_getPadding_with_valid_new_name(self):
+        ret = SignatureScheme.getPadding('rsa_pss_pss_sha256')
```

Comment:
Not sure how complete this should be rsa_pss_rsae_sha384 and 512 seem to be missing. The rsa_pss_sha512 is confusing in the presence of the other options.